src/features/marketplace/store/marketplace.actions.ts

```diff
@@ -141,6 +143,7 @@ export const buyItem = createAsyncThunk(
 
       dispatch(openToastSuccess('The NFT has been bought successfully'));
```

Comment:
imho, we could rephrase this message for something like 'The NFT was successfully purchased.' idk, I find 'bought' not so natural. Thoughts?